.github/workflows/releases.yml

```diff
@@ -6,6 +6,12 @@ on:
     tags:
     - 'v*'
 
+env:
+  REGISTRY: ghcr.io
+  REGISTRY_USER: ${{ github.actor }}
+  REGISTRY_PASSWORD: ${{ github.token }}
+  IMAGE_NAME: ${{ github.repository }}
+
 jobs:
   main:
```

Comment:
I think you're missing a `permissions` attribute here which gives the `packages: write` and `contents: read` permissions?

Reply:
Interesting, Github permissions always confuse me. The build worked on my fork, but maybe thats because it was on main. I'll add those perms.

Reply:
It looks like there's "permissive" and "restricted" permission sets for GitHub Action Tokens:

https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Reply:
Ahh thanks, that makes more sense.